### PR TITLE
Ansible 2.9 tests

### PR DESCRIPTION
**Pull request by felixfontein**
##### SUMMARY
Add CI with Ansible 2.9. Not all platforms are tested; a few should be enough to test that everything works.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
CI

### CODE REVIEW

File: plugins/module_utils/crypto.py [hunk 1]
```diff
@@ -418,7 +418,13 @@ def write_file(module, content, default_mode=None, path=None):
     Uses file arguments from module.
     '''
     # Find out parameters for file
-    file_args = module.load_file_common_arguments(module.params, path=path)
+    try:
```
Comment (gundalow) [inline plugins/module_utils/crypto.py]: Nice that this found a bug straight away :)

Comment (felixfontein) [reply plugins/module_utils/crypto.py]: It's not really a bug, it's using a feature which is only available in Ansible 2.10 :)

**Comment by felixfontein** [comment]
Now I finally managed to work around a (or some) bug in 2.9's ansible-test (which are already fixed in devel and should get backported soon), and to make the workaround for the Ansible 2.10 feature working.

@gundalow is it OK to have six more CI nodes for testing with Ansible 2.9?

**Comment by gundalow** [comment]
> Now I finally managed to work around a (or some) bug in 2.9's ansible-test (which are already fixed in devel and should get backported soon), and to make the workaround for the Ansible 2.10 feature working.
> 
> @gundalow is it OK to have six more CI nodes for testing with Ansible 2.9?

Yes that's fine. Thanks for checking. I've made a note in the GH collections project board to review Shippable usage and Ansible-tests change detection.

**Comment by felixfontein** [comment]
@gundalow thanks! :)

File: plugins/module_utils/crypto.py (hunk 1 above)
Comment (felixfontein) [reply plugins/module_utils/crypto.py:421]: (For reference, it was added in ansible/ansible#61659.)

**Pull request merged**